### PR TITLE
Bugfix - slstr not yet properly supported

### PR DESCRIPTION
**Pull request by adybbroe**
A very rudimentary implementation for the SLSTR instrument was implemented, however, a bug (not providing the maximum scan angle) results in failed tests, see #91 for instance!

The SLSTR instrument is not a cross track scanner, and it seems misleading to have it implemented as if it was. We need to make a more adequate implementation, rather than define an artificial "maximum scan angle" to make it look like an AVHRR.

Until then I propose to remove its "support" and raise an exception.

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [ ] Tests passed 
 - [ ] Passes ``git diff origin/main **/*py | flake8 --diff`` 
 - [ ] Fully documented

### CODE REVIEW

**Comment by adybbroe** [comment]
This was the kind of error I got in the unittests offline in main!
```
        elif instrument.startswith("slstr"):
            instrument = "slstr"
        elif overpass.satellite.name.startswith("aws"):
            scan_angle = 55.25
            instrument = "avhrr"
        else:
            scan_angle = 55.25
>       return instrument, scan_angle
E       UnboundLocalError: cannot access local variable 'scan_angle' where it is not associated with a value

trollsched/boundary.py:88: UnboundLocalError
```

**Comment by adybbroe** [comment]
@pnuu @mraspaud Any opinion on this one?

File: trollsched/boundary.py [hunk 1]
```diff
@@ -38,6 +38,10 @@
               "avhrr-3": "avhrr",
               "mwhs-2": "mwhs2"}
 
+class InstrumentNotSupported(Exception):
+    """Exception to capture cases when instrument are (yet) not supported."""
+    pass
```
Comment (mraspaud) [inline trollsched/boundary.py:43]: you don't need pass :)

**Review by mraspaud — APPROVED** [review]
LGTM, but didn't we add slstr recently?

File: trollsched/boundary.py (hunk 1 above)
Comment (adybbroe) [reply trollsched/boundary.py]: Fixed

**Pull request merged**